uamqp/authentication/cbs_auth.py

```diff
@@ -231,6 +240,8 @@ def __init__(self, audience, uri, token,
                  **kwargs):  # pylint: disable=no-member
         self._retry_policy = retry_policy
         self._encoding = encoding
+        self._override_token_refresh_window = kwargs.pop("override_token_refresh_window", 0)
```

Comment:
I don't think we need to call this "override_token_refresh_window" - _all_ the kwargs are overriding _something_.....
Maybe lets just call it 'refresh_window' throughout?

Reply:
you're 100% right, suggestion taken!